### ISSUE TITLE
Disconnected from native messaging host. Exceeding message byte limit.

### ISSUE THREAD

**Issue by dyce** [Frewacom/pywalfox#65]
First of all, thanks for this super cool project! Unfortunately at some point it just stopped working for me. 

Addon debugging in FF shows me this error in the console: 
`Disconnected from native messaging host: Error: Native application tried to send a message of 1734439797 bytes, which exceeds the limit of 1048576 bytes.`
![2021-05-31-164250_1277x358_scrot](https://user-images.githubusercontent.com/3462328/120231228-3eff7280-c251-11eb-94f5-9c9f135217c3.png)

The debugging section on the addon page shows the following:
![2021-05-31-204709_379x197_scrot](https://user-images.githubusercontent.com/3462328/120232397-cbab3000-c253-11eb-94e0-c4c93455b4d3.png)

Clicking the `Fetch Pywal Colors` seems to do nothing anymore.

I tried and check all of the suggestions in the troubleshooting section, and also reinstalled firefox and wiped the userprofiles but nothing seems to work. Any idea what could cause this?

Best regards!

**Comment by Frewacom** [comment]
Sorry for the late response. 

Honestly, I am not really sure why this happens. I recall someone having a similar issue about a year ago and eventually it boiled down to the python version (2.7). In this case, I am currently using an even newer python version with no issues whatsoever, so I would assume that this is not the cause of the issue. 

Do you know if this started happening after you updated the messaging host? I see that you are using `2.7.2`, which was released quite recently.

**Comment by dyce** [comment]
Don't worry about response time! 

I read through that issue from the other user and wiping Firefox seemed to help there, but that didn't fix it for me. I also tried to install another version of Firefox (like developer or nightly) with a new profile but it was the same problem there.

Unfortunately i don't really remember when it stopped working, but it has to be a some weeks ago at least.

**Comment by Frewacom** [comment]
Perhaps you could try to install a previous version of the messaging host, e.g. `2.7.1` or `2.7.0`, just to make sure?

**Comment by dyce** [comment]
Good idea!  So last working version for me that i can install with `pip` is `2.6`

**Comment by Frewacom** [comment]
Any update on this?

**Comment by dyce** [comment]
`2.6` is working for me, if i switch to `2.7.0 ` the error occurs again.

**Comment by Frewacom** [comment]
Okay, good to know. Thanks!

I will look at this later today 👍🏼
